### ISSUE TITLE
Respect window controls position in Fedora 

### ISSUE THREAD

**Issue by Movisivi** [Taiko2k/TauonMusicBox#579]
This app is the only one that has its controls in the typical top right corner. Anything to do to fix that?

**Comment by Taiko2k** [comment]
No, but I might try experimenting with allowing controls on the left. I'll update this issue if i make any progress on that.

**Comment by Movisivi** [comment]
I found out you can do this   by unchecking  draw own window decorations. That'll work for now until you can make it more good looking

**Comment by Movisivi** [comment]
Looks good but minimize and maximize actions are reversed. Close works fine. I'm on Wayland as well
